wis2box_api/wis2box/env.py

```diff
@@ -25,6 +25,8 @@
 WIS2BOX_URL = os.environ.get('WIS2BOX_URL')
 WIS2BOX_API_URL = os.environ.get('WIS2BOX_API_URL')
 
+WIS2BOX_OBSERVATION_DISTANCE_THRESHOLD = os.environ.get('WIS2BOX_OBSERVATION_DISTANCE_THRESHOLD', '10.0')  # in meters # noqa
```

Comment:
This seems overly restrictive, how well know are the station locations? 100 m, 1km or even 10 km may be better.